x/emissions/migrations/v3/migrate.go

```diff
@@ -86,11 +93,184 @@ func MigrateParams(store storetypes.KVStore, cdc codec.BinaryCodec) error {
 		HalfMaxProcessStakeRemovalsEndBlock: oldParams.HalfMaxProcessStakeRemovalsEndBlock,
 		EpsilonSafeDiv:                      oldParams.EpsilonSafeDiv,
 		DataSendingFee:                      oldParams.DataSendingFee,
-		// NEW PARAMS
-		MaxElementsPerForecast: defaultParams.MaxElementsPerForecast,
+		MaxElementsPerForecast:              defaultParams.MaxElementsPerForecast,
+		MaxActiveTopicsPerBlock:             defaultParams.MaxActiveTopicsPerBlock,
 	}
 
 	store.Delete(types.ParamsKey)
 	store.Set(types.ParamsKey, cdc.MustMarshal(&newParams))
 	return nil
 }
+
+func MigrateActiveTopics(store storetypes.KVStore, ctx sdk.Context, emissionsKeeper keeper.Keeper) error {
```

Comment:
I may be mistaken but technically this wasn't necessary because after the migration, as we talked about, we could simply add even the slightest funding to each of the topics, right? Then the rest of your code would just make it all work. Given that this migration handler is all already here though, we'll keep it -- you've saved us a few steps.